pkg/cli/verifier/envoy_config.go

```diff
@@ -331,10 +331,10 @@ func findOutboundFilterChainForServicePort(meshSvc service.MeshService, dstIPRan
 func getFilterForProtocol(protocol string) string {
 	switch protocol {
 	case constants.ProtocolHTTP:
-		return envoy.HTTPConnectionManagerFilterName
+		return wellknown.HTTPConnectionManager
```

Comment:
@shashankram removed wellknown types in #4834 because they were deprecated; could you link to Envoy docs containing justification for bringing them back?

Reply:
if https://github.com/envoyproxy/go-control-plane/pull/589 gets in then i can close this